correctiontypes.md

```diff
@@ -0,0 +1,132 @@
+# Correction Type Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint contains the various types of corrections that authorized DSpace Users can suggest via the Quality Assurance Framework. The DSpace Users Corrections are indeed one of the possible [Quality Assurence Source](qualityassurancesources.md). 
+
+## Main Endpoint
+**/api/config/correctiontypes**
```

Comment:
This is missing docs on search methods like:
`/api/config/correctiontypes/search/findByItem?uuid=[:uuid]`

Reply:
it is below in the file https://github.com/DSpace/RestContract/pull/250/files#diff-c68780f534af080161c608fed13364eacf2ae9b93fa28e707cc1efa0d45a3cf0R85